modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,46 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a project along with details, such as the namespace in which the Pipeline was created, the last PipelineRun, the status of the Tasks in the PipelineRun, the status of the PipelineRun, and the time taken for the run.
+
+[Discrete]
+.Procedure
+. In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down to see the Pipelines in that project.
++
+//image::op-pipeline-list.png[Pipelines in the Developer perspective]
++
+. Click on the required Pipeline to see the *Pipeline Details* page. It provides a visual representation of all the serial and parallel Tasks in the Pipeline. The Tasks are also listed at the bottom-right of the page. You can click the listed *Tasks* to view the details of the Task.
++
+//image::op-pipeline-details.png[Pipeline details]
++
+. Optionally, in the *Pipeline Details* page:
+
+* Click *Pipeline Runs* tab to see the completed, running, or failed runs for the Pipeline. You can use the Options menu {kebab} to stop a running Pipeline, rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or delete a PipelineRun.
+* Click *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
+* Click *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
+
+////
+. Click *Actions* and select *Start* to start the Pipeline. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the Pipeline definition.
+.. In the *Parameters* section, enter a unique name in the `IMAGE_NAME` field.
+.. Use the drop-downs to select or create the required resources to customize the Pipeline as required. For this example, retain their default values and click *Start* to start the PipelineRun. After the Pipeline starts, the Tasks are executed sequentially in the Pipeline.
+//Specifically for this Pipeline `IMAGE_NAME` may end up as 'APP_NAME' to fulfil what they are looking for. Ticket: https://issues.redhat.com/browse/SRVKP-631 Update docs based on the bug resolution
+
+. The *Pipeline Run Details* page displays the pipeline being executed. You can:
+
+* Hover over the tasks to see the time taken for the execution of each step.
+* Click on a task to see logs for each of the steps in the task.
+* Click the *Logs* tab to see the logs as per the execution sequence of the tasks and use the *Download* button to download the logs to a text file.
```

Comment:
s/as per/according to/
**as per** _prep_
Do not use. Use a phrase such as “according to,” “as,” or “as in.”

Reply:
done